lib/test/cdo/test_i18n_string_url_tracker.rb

```diff
@@ -311,9 +311,13 @@ def self.bench_range
     [1, 1, 1_000, 1_000, 2_000, 5_000, 10_000]
   end
 
-  def bench_linear_performance
-    assert_performance_linear(0.95) do |n|
-      n.times {|m| I18nStringUrlTracker.instance.log(n.to_s, m.to_s, m.to_s, [], '.')}
-    end
-  end
+  # Tests that the tracker's performance scales linearly as the data increases.
```

Comment:
Is it possible to use `skip` here? Example: https://github.com/code-dot-org/code-dot-org/blob/77be800f9d9ac63e123625abfe855a984f289df9/dashboard/test/controllers/home_controller_test.rb#L188. It may be just a Rails thing.

Reply:
yeah you can put `skip` as the first line under the test def and it won't run it

Reply:
oh nice, I didn't know about this.